<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConverterDialog</class>
 <widget class="QDialog" name="ConverterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConverterDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QTextEdit" name="from_edit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="from_label">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="to_label">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QTextEdit" name="to_edit"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="revert_button">
     <property name="text">
      <string>Revert</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="convert_button">
     <property name="text">
      <string>Convert</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
